scripts/cam/plots/exevs_href_precip_spatial_plots.sh

```diff
@@ -72,12 +72,11 @@ for model in $model_list ; do
   for fhr in 24 48 ; do
     past=`$NDATE -$fhr ${VDATE}12`
     INITDATE=${past:0:8}    
-    mv ${model}.v${VDATE}12.0${fhr}h.${mask}.png  evs.${model}.spatial_map.apcp_a24.init${INITDATE}.vlid${VDATE}12.f${fhr}.buk_${mask}.png
+    mv ${model}.v${VDATE}12.0${fhr}h.${mask}.png  evs.${model}.spatial_map.apcp_a24.vlid12z.f${fhr}.buk_${mask}.png
```

Comment:
for spatial maps, we're looking for filenames that match: ${model}.v${VDATE}12.${fhr}.${mask}.gif

$fhr should be a 3-digit value (024, 036, 048)
$mask should just be "conus" and not "buk_conus"

These updates will help streamline Mallory's development of the QPF vs. QPE webpage